.travis/install.sh

```diff
@@ -34,10 +34,12 @@ cd ${TRAVIS_BUILD_DIR}
 mkdir build
 cd build
 
-CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_CXX_COMPILER=$COMPILER"
+#Debug Build Configuration
+CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_CXX_COMPILER=${COMPILER}"
 CMAKE_OPTIONS="$CMAKE_OPTIONS -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME}"
 CMAKE_OPTIONS="$CMAKE_OPTIONS -DGSL_ROOT_DIR=${GSL_ROOT_DIR}"
 CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_CXX_FLAGS_DEBUG=${CMAKE_CXX_FLAGS_DEBUG} -O1"
+CMAKE_OPTIONS="$CMAKE_OPTIONS -DCMAKE_BUILD_TYPE=Debug"
```

Comment:
You could consider building with `RelWithDebInfo` instead of `Debug`, which should speed up the tests and avoid stalls, which is what's causing two of the three failures in Travis.

Reply:
I mostly wanted to build debug to keep running the assert() methods regularly, I'll double check the optimization level is being set correctly before taking out debug.